odoo-shippable/scripts/build-image.sh

```diff
@@ -129,6 +129,18 @@ wget -q -O /usr/share/vim/vim74/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/ru
 sed -i 's/ set mouse\=a/\"set mouse\=a/g' ~/.vimrc
 sed -i "s/let g:neocomplete#enable_at_startup = 1/let g:neocomplete#enable_at_startup = 0/g" ~/.vimrc
 
+# Install YouCompleteMe
+apt-get install cmake
```

Comment:
In my experience, YouCompleteMe is a lot heavier (RAM and disk) than neocomplete. Since Vauxoo mainly writes python code, a lot of YCM's "bulkiness" (--clang-completer, --gocode-completer, --tern-completer, etc) won't be used in practical use cases.

In any case, if you are planning to install YCM, my advice is to uninstall neocomplete.